### DISCUSSION TITLE
Image Preview Zoom toggle setting is gone? Images always zoomed-to-fit on Preview click.

### DISCUSSION THREAD

**Discussion by TKS4** [vladmandic/sdnext#664] [Q&A] [answered]
I could have sworn there was an option in the setting somewhere, but it seems to be gone now. But anytime I click the image to preview it - it defaults to the zoomed view. Does anyone know where the setting is to set the default to non-zoomed view? (so if it's generated a 512x512 image, previews show that, and not some zoom-to-fit it does now). Because currently if I generate a bunch of images and I'm scrolling through, I'm getting a bunch of zoomed blurry images.

**Comment by vladmandic** [comment]
yes. i don't see a reason this should be a server-side setting. i plan to add such functionality to lightbox itself as soon as possible.

**Reply by TKS4** [reply]
I'm a bit confused. The setting was server side in the past? That's weird. At any rate, I thought I might try setting the "js_modal_lightbox_initially_zoomed" in the config.json file to "false" as it was set to true. But it had no effect. Do you know what that setting is for exactly if not for the functionality I was mentioning? Just curious at the moment.

**Reply by vladmandic** [reply]
that setting is ignored. yes, setting was server side in the past and i want to make improvements to it. there is no reason why it should be server-side. give me just a bit of time to make client-side improvements to lightbox.

**Reply by TKS4** [reply]
No problem at all, you take all the time you need! Thank for your help and answering my questions.

**Reply by vladmandic** [reply, marked as answer]
fyi, lightbox zoom will now be whatever you used last. if you open modal lightbox view and zoom image and close it, next image you open will be zoomed by default (and opposite)

**Reply by TKS4** [reply]
Awesome!

**Reply by vladmandic** [reply]
imo, much more easier and intuitive than having global server-side setting.

**Reply by TKS4** [reply]
Not sure if it's easier, I'm still baffled on who, or why such a function was originally server-side at all. Seems dumb, but meh, what do I know.

**Reply by vladmandic** [reply]
i'm trying to a) move as many command line flags to options and b) remove unnecessary server-side options
